### PR TITLE
Fix #99, Guide format fix

### LINKED ISSUE #99: Fix Markdown for Ground System Guide
**Is your feature request related to a problem? Please describe.**
Multiple numbering and formatting inconsistencies in Guide-GroundSystem.md.
Duplicate file with Guide-GroundSystem.txt

**Describe the solution you'd like**
Remove Guide-GroundSystem.txt and update Markdown file using [remark](https://github.com/remarkjs/remark), [prettier](https://prettier.io/) or [Tidy Markdown](https://github.com/notslang/tidy-markdown)

Long-term create a implement style guide for Markdown

Rename file to follow CamelCase.

**Describe alternatives you've considered**
Short manual scrub to fix numbering and code blocks.

### PR DESCRIPTION
**Pull request by skliper**
**Describe the contribution**
Fix #99 - deletes old text file and fixes formatting of markdown

**Testing performed**
Viewed in GitHub.  No code changes.

**Expected behavior changes**
Displays guide as expected.

**System(s) tested on**
GitHub

**Additional context**
None

**Third party code**
None

**Contributor Info - All information REQUIRED for consideration of pull request**
Jacob Hageman - NASA/GSFC

### CODE REVIEW

File: Guide-GroundSystem.md [hunk 1]
```diff
@@ -90,61 +94,66 @@ After completing these steps, restart the Ground System and the changes should h
...
+$ echo /usr/local/lib > /etc/ld.so.conf.d/local.conf
+$ ldconfig
+$ pip install pyzmq
 ```
 #### Install pyQT4 ####
 ```
-
-$ yum install -y PyQt4 $ yum install -y qt qt-demos qt-designer qt4 qt4-designer
-
+$ yum install -y PyQt4
```
Comment (astrogeco) [inline Guide-GroundSystem.md:153]: Use pip3 install for PyQt and point it at PyQt5

File: Guide-GroundSystem.md [hunk 2]
```diff
@@ -90,61 +94,66 @@ After completing these steps, restart the Ground System and the changes should h
...
+$ yum install -y qt qt-demos qt-designer qt4 qt4-designer
```
Comment (astrogeco) [inline Guide-GroundSystem.md:154]: Use qt5 and qt5-designer.

File: Guide-GroundSystem.md [hunk 3]
```diff
@@ -90,61 +94,66 @@ After completing these steps, restart the Ground System and the changes should h
...
-
-$ yum install -y uuid-devel $ yum install -y pkgconfig $ yum install -y libtool $ yum install -y gcc-c++ $ wget <http://download.zeromq.org/zeromq-4.0.5.tar.gz> $ tar xzvf zeromq-4.0.5.tar.gz $ cd zeromq-4.0.5 $ ./configure $ make $ make install $ echo /usr/local/lib > /etc/ld.so.conf.d/local.conf $ ldconfig $ pip install pyzmq
-
+$ yum install -y uuid-devel
+$ yum install -y pkgconfig
+$ yum install -y libtool
+$ yum install -y gcc-c++
+$ wget <http://download.zeromq.org/zeromq-4.0.5.tar.gz>
+$ tar xzvf zeromq-4.0.5.tar.gz
+$ cd zeromq-4.0.5
```
Comment (astrogeco) [inline Guide-GroundSystem.md:143]: I used `pip3 install zmq`

File: Guide-GroundSystem.md [hunk 4]
```diff
@@ -90,61 +94,66 @@ After completing these steps, restart the Ground System and the changes should h
...
-$ yum install -y PyQt4 $ yum install -y qt qt-demos qt-designer qt4 qt4-designer
-
+$ yum install -y PyQt4
+$ yum install -y qt qt-demos qt-designer qt4 qt4-designer
 ```
 #### Running Ground System ###
 ```
-
-$ python GroundSystem.py ```
+$ python GroundSystem.py
```
Comment (astrogeco) [inline Guide-GroundSystem.md:158]: swap for `python3 GroundSystem.py`

**Comment by astrogeco** [comment]
CCB 2020-07-29, APPROVED

**Comment by skliper** [comment]
All good comments relative to the text, but if we really want to update these historical instructions I'd rather go with a python environment and requirements.txt or whatever other more automated method is appropriate.  Shouldn't have to install by hand.

**Comment by astrogeco** [comment]
> All good comments relative to the text, but if we really want to update these historical instructions I'd rather go with a python environment and requirements.txt or whatever other more automated method is appropriate. Shouldn't have to install by hand.

I'm fine leaving them as is since they're historical and we have the comment that they're outdated.

**Pull request merged**